### ISSUE TITLE
get_max_column returns an int with columns A:Z and a float when there are any columns after Z

### ISSUE THREAD

**Issue by jlasky** [socialpoint-labs/sheetfu#37]
I can replicate this error by having data in columns A:AA, but the problem goes away when there is data in columns A:Z.

```
for column in range(0, self.coordinates.number_of_columns):
..
TypeError: 'float' object cannot be interpreted as an integer
```
This seems to be caused by

`get_max_column()`

which returns an int with columns A:Z and a float when there are any columns after Z

**Comment by jlasky** [comment]
The same problem happens even if the range object only includes the column "AA"

`<Range object AA1:AA2>`

**Comment by luchinke** [comment]
Pending to be resolved: #33

**Comment by jlasky** [comment]
resolved in #33
